lib/plenario2/actions/etl_job_actions.ex

```diff
@@ -9,6 +9,11 @@ defmodule Plenario2.Actions.EtlJobActions do
     |> Repo.insert()
   end
 
+  def create!(meta_id) do
```

Comment:
Sure, but why? Is the match error caught elsewhere?

Reply:
I didn't want to bother matching on the outcome since I'm still not sure the best way to handle errors in the GenServer right now. Wrote that so it would just explode.

Reply:
Okie dokie.